web/src/app/search/page.tsx

```diff
@@ -110,16 +112,16 @@ export default async function Home() {
     console.log(`Failed to fetch chat sessions - ${queryResponse?.text()}`);
   }
 
-  let assistants: Persona[] = initialAssistantsList;
+  let personas: Persona[] = initialAssistantsList;
```

Comment:
nit: pref switching it back. We'll need to move everything over to assistants later, so this will just be one additional thing to transition 😓

Reply:
Fair!